backend/molgenis-emx2-webapi/src/main/java/org/molgenis/emx2/web/MolgenisWebservice.java

```diff
@@ -96,6 +116,39 @@ public static void start(int port) {
     afterAfter(sessionManager::updateSession);
   }
 
+  private static Object redirectSchemaToFirstMenuItem(Request request, Response response) {
```

Comment:
why Object , and returning null , when a Route is expected

Reply:
same as previous comment

Reply:
rewrote